solutions/security/ai/use-cases.md

```diff
@@ -18,6 +18,12 @@ The guides in this section describe example workflows for AI Assistant and Attac
 
 For general information, refer to [AI Assistant](/solutions/security/ai/ai-assistant.md) or [Attack discovery](/solutions/security/ai/attack-discovery.md).
 
+## Other AI-powered tools
 
+In addition to AI Assistant and Attack Discovery, {{elastic-sec}} provides several other AI-powered tools for specific use-cases. These include:
+
+* [Automatic Import](/solutions/security/get-started/automatic-import.md): helps you quickly parse, ingest, and create [ECS mappings](https://www.elastic.co/elasticsearch/common-schema) for data from sources that don’t yet have prebuilt Elastic integrations. This can accelerate your migration to {{elastic-sec}}, and help you quickly add new data sources to an existing SIEM solution in {{elastic-sec}}.
+* [Automatic Migration](/solutions/security/get-started/automatic-migration.md): helps you quickly convert SIEM rules from the Splunk Processing Language (SPL) to the Elasticsearch Query Language ({{esql}}). If comparable Elastic-authored rules exist, it simplifies onboarding by mapping your rules to them. Otherwise, it creates custom rules on the fly so you can verify and edit them instead of writing them from scratch.
```

Comment:
```suggestion
* [Automatic Migration](/solutions/security/get-started/automatic-migration.md): Helps you quickly convert SIEM rules from the Splunk Processing Language (SPL) to the Elasticsearch Query Language ({{esql}}). If comparable Elastic-authored rules exist, it simplifies onboarding by mapping your rules to them. Otherwise, it creates custom rules on the fly so you can verify and edit them instead of writing them from scratch.
```